### PR TITLE
Dtm 15523 update dependencies

### PR DESCRIPTION
**Pull request by brenthosie**

### CODE REVIEW

File: sandbox/src/index.js [hunk 1]
```diff
@@ -10,14 +10,17 @@
  * governing permissions and limitations under the License.
  ****************************************************************************************/
 
-import { loadIframe, setDebug } from '../../src/parent';
-
-setDebug(true);
```
Comment (dompuiu) [inline sandbox/src/index.js:15]: Is it ok to remove the line 15?

**Review by dompuiu — APPROVED** [review]

File: sandbox/src/index.js (hunk 1 above)
Comment (brenthosie) [reply sandbox/src/index.js:15]: Yes. The tests were not evaluated when `v11.0.0` was released. `debug` must now be passed in as a part of the `loadIframe` call that returns the bridge. You can see I added that [here](https://github.com/adobe/reactor-bridge/pull/16/files/56298111a214dcc6abb200610e6d065cbcacdff3#diff-58d6e6b1753ada8a39a66c0eb83b7c27e33157cd469c98ab5a2de92c057a9db0R23)

Comment (dompuiu) [reply sandbox/src/index.js:15]: Ok. Thanks!

File: scripts/utils/buildChild.js [hunk 2]
```diff
@@ -15,19 +15,21 @@
...
 const banner = fs.readFileSync('./copyrightBanner.txt', 'utf8');
 
 const plugins = [
-  commonjs(),
+  commonjs({transformMixedEsModules:true}),
   resolve(),
-  babel(),
+  babel({
+    babelHelpers: 'bundled'
+  }),
```
Comment (Aaronius) [inline scripts/utils/buildChild.js:32]: You may have already done this already, but comparing the code in a child built using the new configuration with the code in a child built using the old configuration may be useful in figuring out what kind of bloat these changes may have introduced. I'm not really familiar with how some of these new options affect the output.

**Review by Aaronius — APPROVED** [review]

File: sandbox/src/index.js (hunk 1 above)
Comment (Aaronius) [reply sandbox/src/index.js:15]: Yeah, this is my fault. I never updated the tests previously.

Comment (brenthosie) [reply sandbox/src/index.js:15]: No worries @Aaronius . Thank you for helping me understand `penpal` and the tests here.

File: scripts/utils/buildChild.js (hunk 2 above)
Comment (brenthosie) [reply scripts/utils/buildChild.js:32]: Screen Shot 2021-01-13 at 2 55 22 PM

The `lib` and `dist` directories are similar on both branches 👍

**Pull request merged**